docs/home/whats-new-2-17.md

```diff
@@ -0,0 +1,134 @@
+  
+  !!! Note
+      This feature is only applicable if the all the clusters are version 2.17 and above.
+
+#### Monitoring and Analytics
+
+* <!-- RUN-12901/RUN-16507 - Dashboard improvements MVP -->New GPU Overview dashboard that provides rich and extensive GPU allocation and performance data. The GPU Overview dashboard now contains interactive tiles that provide direct links to the *Nodes*, *Workloads*, and *Departments* tables. Hover over tiles with graphs to show rich data in the selected time frame filter. Tiles with graphs can be downloaded as CSV files. The new dashboard is enabled by default. Use the *Go back to legacy view* to return to the previous dashboard style. For more information, see [Dashboard analysis](../admin/admin-ui-setup/dashboard-analysis.md#overview-dashboard).
+
+* <!-- RUN-15878/RUN-16796/RUN-15878/RUN-16085 omit Knative metrics and update supported metrics and hpa autoscaler -->Updated the knative and autoscaler metrics. Run:ai currently supports the following metrics:
+
+    * Throughput
+    * Concurrency
+  
+  For more information, see [Autoscaling metrics](../admin/workloads/inference-overview.md#autoscaling).
+  
+* <!-- RUN-11488/RUN-17720 deprecation of direct metrics in favor of API -->Improved functionality of metrics by using Run:ai APIs. Using the API endpoints is more efficient and provides an easier way of retrieving metrics in any application. For more information, see [Metrics](../developer/metrics/metrics.md#metrics-apis).
+
+#### Authentication and Authorization
+
+* <!-- RUN-15585/RUN-15586 -  View & Edit SSO settings - SAML -->Added new functionality to SAML 2 SSO configuration to assist with troubleshooting login or permissions issues that may arise. Now administrators have the ability to:
+
+    * View and edit SSO settings.
+    * Upload or download a settings XML file.
+
+For more information, see [SSO UI configuration](../admin/runai-setup/authentication/sso.md#step-1-ui-configuration).
+
+#### Policies
+
+### Control and Visibility
+
+## Deprecation Notifications
+
+Deprecation notifications allow you to plan for future changes in the Run:ai Platform. Deprecated features will be available for **two** versions ahead of the notification. For questions, see your Run:ai representative.
```

Comment:
@jasonnovichRunAI we need to make sure the "two" doesn't refer to the APIs part.... maybe we shouldn't mention a timeframe in this sentence?